pkg/util/util.go

```diff
@@ -176,6 +176,16 @@ func FindItem(prefix string, args []string) string {
 	return ""
 }
 
+// ReplaceArgument replace argument value with given value.
+func ReplaceArgument(prefix string, newValue string, args []string) {
+	for argIndex, arg := range args {
```

Comment:
What happens if `prefix` isn't found? And when multiple prefixes exist? You'll need tests to assert those cases.

This method could return a `bool`, signaling whether there was a replacement.